analysers/analyser_osmosis_highway_deadend.py

```diff
@@ -442,8 +447,16 @@ def __init__(self, config, logger = None):
             resource = 'https://wiki.openstreetmap.org/wiki/Tag:service%3Ddrive-through')
 
         self.callback21 = lambda res: {"class": 1, "data": [self.way_full, self.node_full, self.positionAsText]}
-        self.callback22 = lambda res: {"class": 2, "data": [self.way_full, self.node_full, self.positionAsText]}
-
+        self.callback22 = lambda res: {
+            "class": 2,
+            "data": [self.way_full, self.node_full, self.positionAsText],
+            "text": {"en": "Unconnected highway: {}".format(res[3])}
+}
```

Comment:
I think some indentation is missing here by the way? Same at line 459. There's zero spaces at the `}` while the callback function is indented by ~8 spaces

Reply:
lint doesn't like it indented

Reply:
I think lint won't complain if it's at the same indent as the s from self.callback*